src/lib.rs

```diff
@@ -8,7 +8,12 @@
 // at https://www.apache.org/licenses/LICENSE-2.0 and a copy of the MIT license
 // at https://opensource.org/licenses/MIT.
 
-use std::{hash::{BuildHasherDefault, Hasher}, marker::PhantomData};
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "std")]
+extern crate core;
```

Comment:
We do not need this `extern crate core;` I think.

Reply:
It doesn't compile without.

Reply:
Sorry,  the `Cargo.toml` needs an `edition = "2018"`.

Reply:
Done